### PR TITLE
Plugins: Plugin details right panel improvements

### LINKED ISSUE #93301: Text panel doesn't expand global variables
### What happened?

I created a Text Panel.  when I try to put this in panel:

```
```
${__dashboard}
${__from}
${__to}
${__interval}
${__interval_ms}
${__name}

${__org}
${__user}
${__range}
${__rate_interval}
${__rate_interval_ms}
${timeFilter}
${__timeFilter}
${__timezone}
  ```
```

the Variables ${__dashboard}, ${__from}, ${__to}, ${__org}, ${__user} expand nicely however the other global variables do not.

I'm expecting all variables to expand based on what the dashboard is rendering with.

### What did you expect to happen?

show all variables in expanded form

### Did this work before?

I'm not sure first time I'm trying this

### How do we reproduce it?

1. create a text Panel with the text above
2. try to render text panel and see what stays the same which one expand
3. expected all these global variables expand
4. actual some global variables expand some not

### Is the bug inside a dashboard panel?

_No response_

### Environment (with versions)?

Grafana: v8.5
OS: Mac
Browser: chrome (latest)

### Grafana platform?

None

### Datasource(s)?

_No response_

### LINKED ISSUE #93603: [plugin catalog] changelog tab should pull latest info
The overview/readme uses the installed version, which I think is probably logical. But for changelog - given this could be used to determine whether to update or not - we should likely pull the latest information

Screenshots (not a very exciting example!)
Image
Image
Image

### PR DESCRIPTION
**Pull request by Ukochka**
**What is this feature?**
Small improvements for Plugin details right panel:

- Display latest and installed versions
- Display changelog for latest version and not installed
- Fix displaying the signature type for plugin

**Which issue(s) does this PR fix?**:

Fixes #93603

### CODE REVIEW

**Review by xnyo — APPROVED** [review]
LGTM, awesome work! :raised_hands: :rocket:

File: public/app/features/plugins/admin/api.ts [hunk 1]
```diff
@@ -120,8 +120,8 @@ async function getLocalPluginReadme(id: string): Promise<string> {
 
 async function getLocalPluginChangelog(id: string): Promise<string> {
   try {
-    const markdown: string = await getBackendSrv().get(`${API_ROOT}/${id}/markdown/CHANGELOG`);
-    const markdownAsHtml = markdown ? renderMarkdown(markdown) : '';
+    const pluginFromGcom = await getBackendSrv().get(`api/gnet/plugins/${id}?version=latest`);
+    const markdownAsHtml = pluginFromGcom ? renderMarkdown(pluginFromGcom.changelog) : '';
```
Comment (leventebalogh) [inline public/app/features/plugins/admin/api.ts:124]: Hmm, what is going to happen to plugins that are not published to GCOM? (Until now we were displaying the README that was available locally, but I guess after this change they won't show any README?)

Currently what we are doing is that [we are fetching the remote metadata from GCOM](https://github.com/grafana/grafana/blob/8044cb50f17a021a32e7ac0b83cf8d723457a9ae/public/app/features/plugins/admin/api.ts#L18), [then fetching the locally available one](https://github.com/grafana/grafana/blob/8044cb50f17a021a32e7ac0b83cf8d723457a9ae/public/app/features/plugins/admin/api.ts#L25), and only falling back to the GCOM version of the README if the local version is not available. Maybe we could change this logic instead to use the GCOM version by default?

**Review by leventebalogh — CHANGES_REQUESTED** [review]

File: public/app/features/plugins/admin/api.ts (hunk 1 above)
Comment (jackw) [reply public/app/features/plugins/admin/api.ts:124]: It's also worth remembering that on-prem installations can be locked down and not have access to the internet.

Comment (Ukochka) [reply public/app/features/plugins/admin/api.ts]: You are both right. I have deleted this peace from this method. And as @leventebalogh showed we already have this logic, we just used it in not correct way. Changing the order of changelog helped.  `changelog: remote?.changelog || localChangelog,` if for some reason we can not get remote plugin we will show local changelog

**Review by leventebalogh — APPROVED** [review]
LGTM 👍

**Pull request merged**